src/cheri_pte.sail

```diff
@@ -16,23 +16,62 @@ bitfield PTE_Bits : pteAttribs = {
 /*
  * Reserved PTE bits used by extensions on RV64.
  *
- * There are no such bits on RV32/SV32, so we hard-code both CapWrite=1
- * and CapRead=1 (i.e., there is no ability to disable capability loads
- * from or stores to pages in the SV32 MMU).
+ * For capability store the behaviors are as follows.
+ *
+ *   CW  CD  Action
+ *
+ *    0   X    Trap on tagged capability store
+ *    1   0    CAS the PTE to CW=1, CD=1 (i.e., set both bits) or trap
+ *    1   1    Permit tagged capability store
+ *
+ * The CW/CD pair is akin to SVnn's W/D bits.
```

Comment:
The official names are Sv32 etc; the capital V only comes from C macro naming conventions.